CredentialProvider.Microsoft.VSIX/Microsoft.CredentialProvider.swixproj

```diff
@@ -26,7 +26,7 @@
   <ItemGroup>
     <SigningTarget Include="$(OutDir)$(OutputName)"/>
     <FilesToSign Include="@(SigningTarget)">
-      <Authenticode>Vsix</Authenticode>
+      <Authenticode>VsixSHA2</Authenticode>
```

Comment:
What is the effect of this change? Will anything change in the process of updating the plugin vsix in VS?

Reply:
This was something Zach did because signing wasn't working right somehow. I'm just incorporating his change from another branch here.

Reply:
I think these changes are replacing deprecated SHA1-hashed certs with certs using hash algorithms that aren't compromised